fedimint-server/src/consensus/debug.rs

```diff
@@ -23,7 +23,7 @@ fn item_message(item: &ConsensusItem) -> String {
         ConsensusItem::EpochOutcomeSignatureShare(_) => "Outcome Signature".to_string(),
         // TODO: make this nice again
         ConsensusItem::Module(mci) => {
-            format!("Module CI: module={} ci={}", mci.module_key(), mci)
+            format!("Module CI: module={} ci={}", mci.module_instance_id(), mci)
```

Comment:
I think the kind might be useful here

Reply:
We actually don't have `kind` on the instances of module newtypes, and I'm not sure if it's worth adding. In principle I guess `Decoder` could stuff them there, but then the decoder needs to have it ... and so on.